concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/15 Enable Search.md

```diff
@@ -1,55 +1,4 @@
-To allow users to search through SelectBox values, set [searchEnabled](/api-reference/10%20UI%20Components/dxDropDownList/1%20Configuration/searchEnabled.md '/Documentation/ApiReference/UI_Components/dxSelectBox/Configuration/#searchEnabled') to **true**:
-
----
-##### jQuery
-
-    <!-- tab: index.js -->
-    $(function() {
-            // ...
-
-            $("#selectBox").dxSelectBox({
-                // ...
-                searchEnabled: true
-            });
-    });
-
-##### Angular
-
-    <!-- tab: app.component.html -->
-    <dx-select-box ...
-        [searchEnabled]="true">
-    </dx-select-box>
-
-##### Vue
-
-    <!-- tab: App.vue -->
-    <template>
-        <DxSelectBox ...
-            :search-enabled="true"
-        />
-    </template>
-
-    <script>
-        // ...
-    </script>
-
-##### React
-
-    <!-- tab: App.js -->
-    // ...
-
-    class App extends React.Component {
-        render() {
-            return (
-                <SelectBox ...
-                    searchEnabled={true}
-                />
-            );
-        }
-    }
-    export default App;
-
----
+#include tutorials-editors-enable-search
 
 There are additional search properties demonstrated in the following demo:
```

Comment:
Why "there are"? We can safely omit this phrase without any losses.
```suggestion
Additional search properties demonstrated in the following demo:
```

Reply:
But then it is not a valid sentence. It still requires passive: 'Additional search properties are demonstrated in the following demo:' 

However, I would prefer the following: 'This demo shows additional search properties: '

Reply:
Yes, you are correct. And I like your latest version.

```suggestion
This demo shows additional search properties:
```